### ISSUE TITLE
New Bot API 2.0

### ISSUE THREAD

**Issue by fylux** [format37/CalcuBot#1]
Would be great to add new features with the new API.

**Comment by format37** [comment]
What, for example?

**Comment by fylux** [comment]
A GUI for the calculator with scientific and normal mode.

**Comment by fylux** [comment]
You could also consider some kind of combination with LatexBot.

**Comment by format37** [comment]
You are right. It would be nice to add inline feature. Going to do this summer. Still not sure about the GUI. don't want to complicate things. let's see how it will look.

Give an example of a combination with a LatexBot?

**Comment by fylux** [comment]
With LatexBot you could create a new command, /clx that does the same /cl does but instead of returning the result as message it is returned in Latex format.

Greetings.
